app/models/mandaat.js

```diff
@@ -17,4 +17,18 @@ export default class MandaatModel extends Post {
     inverse: 'bevat',
   })
   bevatIn;
+
+  async isDecretaalHelper() {
+    const bestuursorgaanPositions = await this.bevatIn;
+    const areDecretaal = await Promise.all(
+      bestuursorgaanPositions.map((position) => position.isDecretaal())
+    );
+
+    return areDecretaal.some(Boolean);
+  }
+
+  // If any of the bestuursorganen is decretaal, then the mandaat is decretaal.
+  get isDecretaal() {
```

Comment:
can a getter not be async? if it can, why is this extra function necessary?

Reply:
it can be. however, when using this in templates, it will be a pain and will require promise helpers or the use of https://api.emberjs.com/ember/release/classes/PromiseProxyMixin. We should consider adding ember-resources here https://github.com/NullVoxPopuli/ember-resources/blob/main/docs/docs/resources.md

Reply:
will add a ticket to examine ember resources, ignore for now and use ember-promise-helpers when accessing this in templates